### ISSUE TITLE
UI.DrawTexture AspectRatio

### ISSUE THREAD

**Issue by zorg93** [crosire/scripthookvdotnet#466]
I know this was put to rest a while ago, but the current system seems broken for when you have a rotated texture and dont use a the GET_SCREEN_ASPECT_RATIO correction. Instead of rotating the sprite, it will also stretch it in the X direction. I didnt want to submit any kind of commit until i hear what people think is best. But this code here seemed to fix rotation issues
Size FixedSize = new Size((int)(Size * Call<float>(Hash._GET_SCREEN_ASPECT_RATIO, 0)), Size);
            UI.DrawTexture(SpeedoArrow2, 0, -9998, 100, Screenloc, new PointF(0.5f, 0.5f), FixedSize, rotation2, Color.White, Call<float>(Hash._GET_SCREEN_ASPECT_RATIO, 0));     

Basically just scaling the X component of the size by the aspect ratio and using the aspect ratio in the DrawTexture call

**Comment by crosire** [comment]
Sounds good to me.

**Comment by zorg93** [comment]
Discuss in #467
